lora/lorawan/otaa.go

```diff
@@ -4,16 +4,15 @@ import (
 	"bytes"
 	"crypto/aes"
 	"encoding/hex"
-	"errors"
 )
 
 // Otaa is used to store Over The Air Activation data of a LoRaWAN session
 type Otaa struct {
 	DevEUI   [8]uint8
 	AppEUI   [8]uint8
 	AppKey   [16]uint8
-	DevNonce [2]uint8
-	AppNonce [3]uint8
+	devNonce [2]uint8
```

Comment:
You made these two variable private 
Should we do the same in a future PR, and implement Getters when needed ?

Reply:
Good idea!